sample.ipynb

```diff
@@ -78,7 +76,57 @@
    ]
   },
   {
-   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below is a cell containing a magic command, it should be skipped."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%timeit 42 * 42\n",
+    "# lambda would normally be flagged as error but should be skipped\n",
```

Comment:
With the new suggested `pass`, this cell won't be skipped. As I mentioned in the comment, probably best to not change `sample.ipynb` and have these two new cells in a. separate notebook, until we sort out the testing.